contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,108 @@
+---
+date: 2022-08-29
+title: 'Array 1.39.0: Moving person properties onto events'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: PostHog 1.39.0 moves person properties onto events, resulting in queries which are up to 400x faster. And that's just for starters!
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [Upgrading PostHog guide](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.39.0 release notes
+
+**Release highlights:**
+
+- New: Joe to add Persons on Events here
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- New: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Improved: Paul to add a highlight here
+- Apps: JOE to add
+
+### New: Person properties have moved on to events
```

Comment:
@tiina303 @EDsCODE I've added some content in here. We'll link to the full post when it's done.